README.md

```diff
@@ -1,14 +1,24 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-F11-1/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is a sample project for Software Engineering (SE) students.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+**CCACommander Ultra Promax Xtra 9000+** frees your mind of having to remember your CCA members and events. It's,
+* text-based
+* easy to learn
+* SUPER FAST to use
+
+### Features
+1. Add and delete members
+2. Add and delete events
+3. List events and members
```

Comment:
Should we include something about listing events associated to members and members associated to events? Maybe something like:
```suggestion
3. List events and members
3.5 List events of members and members of events
```

Reply:
Updated

Reply:
LGTM, will merge now